docs/guidance/gcs-ft.md

```diff
@@ -80,7 +78,7 @@ You can also specify additional environment variables in the head pod to customi
 
 #### KubeRay Operator Controller
 
-KubeRay Operator controller watches for new `Event` reconcile call. If this Event object is to notify the failed readiness probe,
+KubeRay Operator controller watches for necessary head pod's `Event` reconcile call. If this Event object is to notify the failed readiness probe,
```

Comment:
This is not totally correct. We still watch unnecessary Pod events. See [here](https://github.com/ray-project/kuberay/issues/601#issuecomment-1486149187) for more details.